### PR TITLE
Create game ending screen

### PR DESCRIPTION
**Pull request by glowacki-dev**
Fix: #115 
Fix: #131 
Fix: #173 

It only shows a list of players who are winners and allows quickly joining a new game
image

Additionally the waiting screen has been modified slightly, so it's no longer forced into a grey rectangle
image
image
image
image

### CODE REVIEW

**Comment by pastpatryk** [comment]
Maybe we can add `max-width` to the Pictionary settings form? It looks super weird when it's that wide.

File: src/components/game/GameEndingBoard/index.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  const { playerID, matchID, gameName, credentials } = useBoardGame();
+  const [loading, setLoading] = useState(false);
+
+  const user = useUser();
+  const history = useHistory();
+
+  const handleNewMatch = useCallback(() => {
+    setLoading(true);
+    lobbyClient
+      .playAgain(gameName, matchID, { playerID: playerID, credentials: credentials })
```
Comment (pastpatryk) [inline src/components/game/GameEndingBoard/index.js:27]: Nice 👍

Comment (glowacki-dev) [reply src/components/game/GameEndingBoard/index.js:27]: It's been there since v0.39.10. We must have missed it somehow 😄

**Comment by pastpatryk** [comment]
🤔 although it looks nice I prefer the version with white background. It's more consistent with the rest of the interface

**Comment by glowacki-dev** [comment]
Ok, leaving the white background then

**Review by pastpatryk — APPROVED** [review]

**Pull request merged**